src/components/GalleryLightBox/_gallery-lightbox.scss

```diff
@@ -0,0 +1,207 @@
+// 2019-11-11
+// ----------------------------------
+
+// @import '~pure-react-carousel/dist/react-carousel.es.css';
+
+.cc-gallery-lightbox {
+  background-color: var(--colour-grey-80);
+  border: 0;
+  bottom: 0;
+  color: var(--colour-white);
+  height: 100%;
+  left: 0;
+  padding: var(--space-xl);
+  position: fixed;
+  right: 0;
+  top: 0;
+  width: 100%;
+  z-index: var(--z-modal);
+}
+
+.cc-gallery-lightbox__carousel {
+  height: 100%;
+  position: relative;
+}
+
+.cc-gallery-lightbox__slider {
+  bottom: 0;
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+}
+
+.cc-gallery-lightbox__carousel-tray-wrap,
+.cc-gallery-lightbox__carousel-tray {
+  height: 100%;
+  overflow: hidden;
+}
+
+.cc-gallery-lightbox__carousel-tray {
+  @extend %list-clean;
+
+  display: flex;
+  transition: transform 0.5s;
+  transition-timing-function: ease;
+  transition-timing-function: cubic-bezier(0.645, 0.045, 0.355, 1);
+  will-change: transform;
+}
+
+.cc-gallery-lightbox__slide {
+  flex-basis: 100vw;
+  height: 100%;
+
+  /**
+   * Becuase our carousel library wants to set an aspect-ratio on our slides,
+   * based on the natural height and width of the images, it sets the padding-bottom
+   * property inline to style the slides appropriately.
+   *
+   * We're rolling our own styles, and so we want to override this.
+   */
+  padding-bottom: 0 !important;
+}
+
+.cc-gallery-lightbox__slide-layout {
+  // Reset <figure> style
+  @extend %margin-0;
+
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  overflow: scroll;
+
+  @include mq(sm) {
+    justify-content: space-between;
+  }
+
+  @include mq(md) {
+    flex-direction: row;
+  }
+}
+
+.cc-gallery-lightbox__image-pane {
+  display: flex;
+  flex-basis: 50%;
+  flex-direction: column;
+  flex-grow: 1;
+  overflow: hidden;
+
+  @include mq(sm) {
+    flex-basis: 57%;
+  }
+
+  @include mq(md) {
+    flex-basis: 70%;
+    padding-right: var(--space-lg);
+  }
+}
+
+.cc-gallery-lightbox__image-pane-stage {
+  flex-basis: 90%;
+  margin-bottom: var(--space-lg);
+  overflow: hidden;
+
+  @include mq(sm) {
+    height: auto;
+  }
+
+  @include mq(md) {
+    flex-basis: 84%;
+    margin-bottom: var(--space-lg);
+    overflow: hidden;
+  }
+
+  > img {
+    display: block;
+    height: 100%;
+    object-fit: contain;
+    width: 100%;
+  }
+}
+
+.cc-gallery-lightbox__image-pane-tray {
+  display: block;
+
+  @include mq(md) {
+    flex-basis: 16%;
+  }
+}
+
+.cc-gallery-lightbox__info-pane {
+  flex-basis: 50%;
+  overflow: scroll;
+
+  @include mq(sm) {
+    flex-basis: 43%;
+  }
+
+  @include mq(md) {
+    flex-basis: 30%;
+    padding-left: var(--space-lg);
+  }
+}
+
+.cc-gallery-lightbox__meta {
+  border-left: 1px solid var(--colour-white);
+  font-family: monospace;
+  font-size: var(--body-xs);
+  margin-bottom: var(--space-md);
+  padding-left: var(--space-md);
+}
+
+.cc-gallery-lightbox__meta-item {
+  display: inline;
+
+  // Override styles from wysiwyg
+  > p {
```

Comment:
wysiwyg overrides are an issue for a number of components.

We don't need to fix it across the board in a perfect generic/future proof/#componentLibrariesWillNeverBeBetterThanThis fashion right now - it's something for the future

Reply:
ah - so you're saying I should override this from within corporate-react as opposed to here?

Reply:
No I think corporate-components IS the place to override it.

I've done a similar thing for Text component. There are also text snippets to think about (very like Text paragraph type but much more pared back editing capabilities - basically a Text paragraph for an infobox).

Could this be an application for an object? e.g. `.o-rich-text` as it seems more than just utility

Reply:
ah I get you, I think it would be place for adding a scope style actually as I think it falls more under that category than an object https://csswizardry.com/2015/03/more-transparent-ui-code-with-namespaces/#scope-namespaces-s-